aws/resource_aws_route53_zone.go

```diff
@@ -138,7 +143,7 @@ func resourceAwsRoute53ZoneCreate(d *schema.ResourceData, meta interface{}) erro
 		}
 	}
 
-	if err := keyvaluetags.Route53UpdateTags(conn, d.Id(), route53.TagResourceTypeHostedzone, map[string]interface{}{}, d.Get("tags").(map[string]interface{})); err != nil {
+	if err := keyvaluetags.Route53UpdateTags(conn, d.Id(), route53.TagResourceTypeHostedzone, nil, tags); err != nil {
```

Comment:
similar comment as above to pass in `tags.Map()` instead of `tags`...hmm or maybe we need to adjust the `keyvaluetags.New` method if we find ourselves doing this too often  🤔

Reply:
Submitted https://github.com/hashicorp/terraform-provider-aws/pull/19011 to cover this nonintuitive situation 👍